extensions/mssql/src/features.ts

```diff
@@ -27,6 +29,41 @@ export class TelemetryFeature implements StaticFeature {
 	}
 }
 
+export class AccountFeature implements StaticFeature {
+
+	constructor(private _client: SqlOpsDataClient) { }
+
+	fillClientCapabilities(capabilities: ClientCapabilities): void {
+		Utils.ensure(capabilities, 'telemetry')!.telemetry = true;
+	}
+
+	initialize(): void {
+		this._client.onRequest(contracts.SecurityTokenRequest.type, async e => {
+			const accountList = await azdata.accounts.getAllAccounts();
+
+			if (accountList.length < 1) {
+				// TODO: Prompt user to add account
+				window.showErrorMessage(localize('mssql.missingLinkedAzureAccount', 'Azure Data Studio needs to contact Azure Key Vault to access a column master key for Always Encrypted, but no linked Azure account is available. Please add a linked Azure account and retry the query.'));
+				return null;
+			} else if (accountList.length > 1) {
+				// TODO: Prompt user to select an account
+				window.showErrorMessage(localize('mssql.multipleLinkedAzureAccount', 'Azure Data Studio needs to contact Azure Key Vault to access a column master key for Always Encrypted, which is not supported if multiple linked Azure accounts are present. Make sure only one linked Azure account exists and retry the query.'));
+				return null;
```

Comment:
Use undefined instead of null

Reply:
And what is the behavior if we return undefined here? How does the tools service side handle that?

Reply:
Currently SqlToolsService ignores it. On the front end the user is (termorarily) presented with the message to add an account or use a single account. If not then the query is stopped and is the same as if the query is canceled by the user. We will be adding more for an improved UX in following iterations, which is what the TODOs are denoting.

Reply:
Changed null to undefined as requested.